### DISCUSSION TITLE
Client connection limits

### DISCUSSION THREAD

**Discussion by PedroMPagani** [seaweedfs/seaweedfs#4673] [Q&A]
Is there a client limitation on seaweed by default? on the number of clients that there can be at once?

**Comment by chrislusf** [comment]
What is your use case?

**Reply by PedroMPagani** [reply]
Currently using the Java client to store Minecraft world files, I am just wondering if there is one scenario where Seaweed will call something such as deny a connection if there are many, like the MySQL maxconnection variable. I am creating a FilerClient everytime I need to grab some information, and closing it right after, unsure if that's the best use case.

**Reply by chrislusf** [reply]
Java client is fine. It is just like another http client.

**Reply by PedroMPagani** [reply]
Understood. Just for understanding of the system, the request for a file data goes through the filer client, the filer will grab from the volume and then share with the requester? I am looking at the piece which would be the bandwidth limitation.
